common/scala/src/main/scala/whisk/common/Logging.scala

```diff
@@ -260,6 +260,7 @@ object LoggingMarkers {
   // Time of the activation in controller until it is delivered to Kafka
   val CONTROLLER_ACTIVATION = LogMarkerToken(controller, activation, start)
   val CONTROLLER_ACTIVATION_BLOCKING = LogMarkerToken(controller, "blockingActivation", start)
+  val CONTROLLER_ACTIVATION_BLOCKING_ACTIVATION_STORE = LogMarkerToken(controller, activation, count)
```

Comment:
"controller_activation_count" doesn't feel like the right metric to use here. `controller_activationPoll_count` maybe?